python/ql/src/semmle/python/objects/Modules.qll

```diff
@@ -367,12 +367,10 @@ class AbsentModuleAttributeObjectInternal extends ObjectInternal, TAbsentModuleA
     }
 
     override predicate callResult(ObjectInternal obj, CfgOrigin origin) {
-        // Don't know, assume not callable.
-        none()
+        obj = ObjectInternal::unknown() and origin = CfgOrigin::unknown()
     }
 
     override predicate callResult(PointsToContext callee, ObjectInternal obj, CfgOrigin origin) {
-        // Don't know, assume not callable.
```

Comment:
Given the above change to `callResult`, would it be reasonable to do the same thing here? Or at least it'd be useful to include a note on why it's different.

Reply:
These two predicates should be mutually exclusive.
I've improved the QL doc on the base class to make that clearer.